test_scripts/Resumption/InteriorVehicleData/017_InteriorVD_resumption_two_apps_same_moduleTypes_different_moduleIds_unexpected_disconnect.lua

```diff
@@ -0,0 +1,72 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data for two apps after transport disconnect
+--  in case apps are subscribed to the same moduleTypes with different moduleIds
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app1 and app2 with REMOTE_CONTROL hmi type are registered and activated
+-- 3. App1 is subscribed to module_1 with moduleId_1
+-- 4. App2 is subscribed to module_1 with moduleId_2
+--
+-- Sequence:
+-- 1. Transport disconnect and reconnect are performed
+-- 2. Apps start registration with actual hashIds after unexpected disconnect
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(module_1, moduleId_1) and RC.GetInteriorVehicleData(module_1, moduleId_2)
+--   to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to both mobile apps
```

Comment:
b. SDL get a successful response for every GetInteriorVehicleData request with isSubscribed=true from HMI
c. SDL respond RAI(success=true, result code = SUCCESS) to both mobile apps
d. update hashId after successful resumption

Reply:
Done